### PR TITLE
Chore/gcweb 11.1.2

### PR DESCRIPTION
**Pull request by harrdou**
- Update the GCWeb theme to 11.1.2
    - Implement Subresource Integrity
    - Add the RP name to the page titles as recommended by accessibility testing

### CODE REVIEW

File: gluu-server/opt/gluu/jetty/oxauth/custom/pages/WEB-INF/incl/components/head-en.xhtml [hunk 1]
```diff
@@ -10,12 +10,14 @@
...
     <meta name="dcterms.subject" content="#{title != null ? title : 'Sign In'}" />
     <meta name="dcterms.issued" content="2020-02-21" />
-    <meta name="dcterms.modified" content="2022-01-07" />
+    <meta name="dcterms.modified" content="2022-07-28" />
     <meta name="dcterms.spatial" content="Canada" />
     <meta name="dcterms.type" content="navigation page - topic page" />
     <meta name="dcterms.identifier" content="Sign_In_Canada" />
 
-    <title>#{title != null ? title : "Sign In"}</title>
+    <title>#{title != null ? title : "Sign In to"}
```
Comment (innovatechs) [inline gluu-server/opt/gluu/jetty/oxauth/custom/pages/WEB-INF/incl/components/head-en.xhtml:18]: Space after "Sign In to" ("Sign In to ") ? 
Same for "Se connecter à "

File: gluu-server/opt/gluu/jetty/oxauth/custom/pages/WEB-INF/incl/components/head-fr.xhtml [hunk 2]
```diff
@@ -8,14 +8,16 @@
...
+    <meta name="dcterms.subject" content="#{title != null ? title : 'Se connecter'}" />
     <meta name="dcterms.issued" content="2020-02-21" />
-    <meta name="dcterms.modified" content="2022-01-07" />
+    <meta name="dcterms.modified" content="2022-07-28" />
     <meta name="dcterms.spatial" content="Canada" />
     <meta name="dcterms.type" content="navigation page - topic page" />
     <meta name="dcterms.identifier" content="Authenti_Canada" />
 
-    <title>#{title != null ? title : "Ouvrez une séance"}</title>
+    <title>#{title != null ? title : "Se connecter à"}
```
Comment (innovatechs) [inline gluu-server/opt/gluu/jetty/oxauth/custom/pages/WEB-INF/incl/components/head-fr.xhtml:18]: Missing space "Se connecter à " ?

**Review by innovatechs — COMMENTED** [review]
Missing spaces ?

File: gluu-server/opt/gluu/jetty/oxauth/custom/pages/WEB-INF/incl/components/head-en.xhtml (hunk 1 above)
Comment (harrdou) [reply gluu-server/opt/gluu/jetty/oxauth/custom/pages/WEB-INF/incl/components/head-en.xhtml:18]: There is a newline as whitespace.

Comment (innovatechs) [reply gluu-server/opt/gluu/jetty/oxauth/custom/pages/WEB-INF/incl/components/head-en.xhtml:18]: Ah ok, if that works then great !

**Review by innovatechs — APPROVED** [review]

**Pull request merged**